lib/src/manifest.test.ts

```diff
@@ -146,3 +146,54 @@ describe('manifest file operations', () => {
         }
     });
 });
+
+describe('manifest validation', () => {
+    // Path to our temporary directory and file
+    const tmpDir = join(tmpdir(), 'xpoc_tmp');
+    const tmpFilePath = join(tmpDir, 'testmanifest.json');
```

Comment:
We should leave a TODO near by to create manifests with errors; that'd be useful.